agent_pool.go

```diff
@@ -48,18 +51,21 @@ type AgentPool struct {
 
 	// Relations
 	Organization *Organization `jsonapi:"relation,organization"`
+	Workspaces   []*Workspace  `jsonapi:"relation,workspaces"`
 }
 
 // A list of relations to include
 type AgentPoolIncludeOpt string
 
 const AgentPoolWorkspaces AgentPoolIncludeOpt = "workspaces"
 
+type AgentPoolReadOptions struct {
+	Include []AgentPoolIncludeOpt `url:"include,omitempty"`
+}
+
 // AgentPoolListOptions represents the options for listing agent pools.
 type AgentPoolListOptions struct {
 	ListOptions
-
-	Include []AgentPoolIncludeOpt `url:"include,omitempty"`
```

Comment:
I thought this was possible on the List as well. Why is it removed?

Reply:
It is not possible. I was surprised myself. But my digging how agent pool is implemented in Atlas took me to that conclusion.

Reply:
```
$ curl -s -H "Authorization: Bearer $(tfc_local_token)" https://tfcdev-5839ba17.ngrok.io/api/v2/organizations/hashicorp/agent-pools?include=workspaces | jq .included[0].attributes.name
"example"
```

Reply:
ooo OK! Lemme try that as integration test and see what results I get